### PR TITLE
Add blog post on cycle 5

### PR DESCRIPTION
**Pull request by m52go**
Summarize results and call for more due diligence (leo/lio incident). @erciccione you're also welcome to review...I couldn't assign you as a reviewer.

### CODE REVIEW

**Review by erciccione — APPROVED** [review]
Looks good to me.

ACK

File: _posts/2019-09-18-cycle-5-results.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+It also set the `RECIPIENT_BTC_ADDRESS` DAO parameter (aka "donation address") to the bitcoin address specified in the same GitHub proposal linked above.
+
+Part of the core purpose of the Bisq DAO is to [route trading fees directly from traders to contributors](https://docs.bisq.network/user-dao-intro.html#earn-and-distribute-revenue){:target="_blank"} with no one person or entity authorizing funds, providing funds, or routing funds. This is why we encourage Bisq users to use BSQ to pay trading fees, but we don't coerce, so BSQ is optional. Traders can still pay trading fees in BTC. Instead of going to arbitrators, BTC fees will soon go to the donation address.
+
+The person taking on this role will periodically purchase BSQ with the BTC accumulated in the donation address. They will be be assigned to the [corresponding role issue](https://github.com/bisq-network/roles/issues/80){:target="_blank"} shortly, where you will see monthly updates.
```
Comment (devinbileck) [inline _posts/2019-09-18-cycle-5-results.md]:
```suggestion
The person taking on this role will periodically purchase BSQ with the BTC accumulated in the donation address. They will be assigned to the [corresponding role issue](https://github.com/bisq-network/roles/issues/80){:target="_blank"} shortly, where you will see monthly updates.
```

File: _posts/2019-09-18-cycle-5-results.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+**Create Analytics Operator and Analytics Administrator roles**
+
+_Generic proposal ([link](https://github.com/bisq-network/proposals/issues/115){:target="_blank"})_
+
+Google Analytics was recently replaced with Matomo for better user privacy on Bisq websites.
+
+As a result, new roles were approved to operate the Matomo server and administer it as needed.
+
+**Improve Bisq testing framworks**
```
Comment (devinbileck) [inline _posts/2019-09-18-cycle-5-results.md]:
```suggestion
**Improve Bisq testing frameworks**
```

**Review by devinbileck — APPROVED** [review]
utACK
Looks good! Just a few minor corrections.

**Comment by m52go** [comment]
Thanks for reviewing! I got so wrapped in detailing proposals that I forgot to add a summary of the cycle's time frame and economics. Added those in another commit.

**Review by erciccione — CHANGES_REQUESTED** [review]
Yeah the summary is definitely nice to have. Now the PR only need to be squashed and then will be ok IMO

**Review by erciccione — APPROVED** [review]

**Pull request merged**